### PR TITLE
Time Slider, Map Zoom styles and layout 

### LINKED ISSUE #256: Fix Build
Build is failing:
https://travis-ci.org/GlobalFishingWatch/pelagos-client/builds/153573368

### PR DESCRIPTION
**Pull request by jarmstrongbts**
Connects https://github.com/SkyTruth/GFW-Tasks/issues/256
- Reduce size of sidebar on/off switches
- Move Google Maps zoom and loading controls down
- Make Time slider styles dark blue
- Put play/pause/share/zoom controls on top left of slider (time slider now full-width)

### CODE REVIEW

File: js/app/Visualization/UI/UIManager.js [hunk 1]
```diff
@@ -186,6 +186,11 @@ define([
       self.loadingNode = $('<div class="loading"><img style="width: 20px;" src="' + Paths.LoaderIcon + '"></div>');
       self.visualization.animations.map.controls[google.maps.ControlPosition.LEFT_TOP].push(self.loadingNode[0]);
 
+      /* Add a class to the zoom buttons so we can modify them with our css */
+
+      $(".gm-bundled-control .gmnoprint:has(div[title='Zoom in'])").addClass('zoomButtons')
```
Comment (pwoods25443) [inline js/app/Visualization/UI/UIManager.js:191]: @enriquetuya here is where we add the class to the map zoom buttons.  It is possible that this happens to soon before the buttons have been initialized.

File: js/app/Visualization/UI/style.less [hunk 2]
```diff
@@ -109,26 +109,24 @@ body {
 .loading {
   padding-left: 14px;
   padding-top: 10px;
+  position: absolute;
+  top: 175px !important;
 }
 
+.zoomButtons {
```
Comment (pwoods25443) [inline js/app/Visualization/UI/style.less:116]: @enriquetuya here is where we force the absolute positioning on to the google maps zoom buttons.  Also note the positioning of the  "loading spinner" in the .loading class above

File: js/app/Visualization/UI/SidePanels/SidePanelManager.js [hunk 3]
```diff
@@ -45,7 +45,7 @@ define([
         '        <a id="feedback_url" target="_blank">' +
         '          Feedback' +
         '        </a>' +
-        '        <div id="collapse-button"><img src="%(img)s/buttons/close.png"></div>' +
+        '        <div id="collapse-button"><i class="fa fa-chevron-circle-right"></i><!--img src="%(img)s/buttons/close.png"--></div>' +
```
Comment (enriquetuya) [inline js/app/Visualization/UI/SidePanels/SidePanelManager.js:48]: @jarmstrongbts The following is just a personal opinion, I did not like how this new icon looks like in the menu. It's the most predominant icon when you give a quick look to the menu. What about using `chevron-right` with NO circle for closing?

Comment (enriquetuya) [inline js/app/Visualization/UI/SidePanels/SidePanelManager.js:48]: @jarmstrongbts please remove commented html code.

File: js/app/Visualization/UI/SidePanels/SidePanelManager.js [hunk 4]
```diff
@@ -35,7 +35,7 @@ define([
 
       self.node = $(new ObjectTemplate('' +        
         '<div id="w" class="expanded">' +
-        '  <div id="expand-button"><img src="%(img)s/buttons/open.png"></div>' +
+        '  <div id="expand-button"><i class="fa fa-chevron-circle-left"></i><!--img src="%(img)s/buttons/open.png"--></div>' +
```
Comment (enriquetuya) [inline js/app/Visualization/UI/SidePanels/SidePanelManager.js:38]: @jarmstrongbts please remove commented html code.

File: js/app/Visualization/UI/UIManager.js [hunk 5]
```diff
@@ -101,9 +101,9 @@ define([
 
       self.controlButtonsNode = $(new ObjectTemplate(''
         + '<div class="control_box">'
-        + '  <button class="btn btn-default btn-lg" data-name="share"><i class="fa fa-share"></i></button>'
-        + '  <div class="divide"></div>'        
-        + '  <button class="btn btn-default btn-lg" data-name="play"><i class="fa fa-play paused"></i><i class="fa fa-pause playing"></i><!--<img class="paused" src="%(img)s/buttons/play.png"><img class="playing" src="%(img)s/buttons/pause.png">--></button>'
+        + '  <div><button class="btn btn-default btn-lg" data-name="play"><i title="play" class="fa fa-play paused"></i><i title="pause" class="fa fa-pause playing"></i><!--<img class="paused" src="%(img)s/buttons/play.png"><img class="playing" src="%(img)s/buttons/pause.png">--></button></div>'
+        + '  <!--div class="divide"></div-->'
```
Comment (enriquetuya) [inline js/app/Visualization/UI/UIManager.js:105]: @jarmstrongbts Please remove commented code.

File: js/app/Visualization/UI/UIManager.js (hunk 1 above)
Comment (enriquetuya) [reply js/app/Visualization/UI/UIManager.js:191]: Added new ticket for @redhog to work on this tomorrow morning: https://github.com/SkyTruth/pelagos-server/issues/1128

File: js/app/Visualization/UI/Widgets/Timeline/Timeline.less [hunk 6]
```diff
@@ -196,7 +208,7 @@
         width: 100%;
 
         white-space: nowrap;
-        font-size: 0;
+        font-size: 10pt;
```
Comment (redhog) [inline js/app/Visualization/UI/Widgets/Timeline/Timeline.less:211]: @jarmstrongbts This looks a bit dangerous. font-size: 0 is generally a trick for box sizing. Why did you change this?

**Pull request merged**